cherrypy/_cplogging.py

```diff
@@ -126,12 +128,15 @@ class NullHandler(logging.Handler):
     """A no-op logging handler to silence the logging.lastResort handler."""
 
     def handle(self, record):
+        """Handle stub."""
         pass
 
     def emit(self, record):
+        """Emit stub."""
```

Comment:
In general, the docstrings should be more informative (except for some dunder methods).

This probably emits a record, not a stub, by the way.

Reply:
```suggestion
        """Emit  a log record doing no-op."""
```